### PR TITLE
MAILBOX-328 Avoid tables of more than 30 characters

### PR DESCRIPTION
**Pull request by chibenwa**
For compatibility with Oracle 11g database.

Please note procedure for migration and associated communication plan is in the message commit of commit https://github.com/linagora/james-project/commit/babb744e0148b3eb1269cb0f8ad2f521acc449b2

This simple to fix, user reported problem was discussed as needed with Raphael Ouazana.

### CODE REVIEW

File: mailbox/jpa/src/main/java/org/apache/james/mailbox/jpa/quota/model/MaxDomainMessageCount.java [hunk 1]
```diff
@@ -27,7 +27,7 @@
 import org.apache.james.core.Domain;
 
 @Entity(name = "MaxDomainMessageCount")
-@Table(name = "JAMES_MAX_DOMAIN_MESSAGE_COUNT")
```
Comment (mbaechler) [inline mailbox/jpa/src/main/java/org/apache/james/mailbox/jpa/quota/model/MaxDomainMessageCount.java]: we removed an interesting part, why don't we remove "JAMES_" prefix instead ?

**Comment by mbaechler** [comment]
test this please

**Comment by chibenwa** [comment]
To sum up offline discussions, we still have no migration strategy (other than manual user intervention).

**Comment by aduprat** [comment]
Build triggered. sha1 is original commit.

**Comment by aduprat** [comment]
Build triggered. sha1 is original commit.

**Comment by aduprat** [comment]
Build triggered. sha1 is original commit.

**Comment by aduprat** [comment]
Build triggered. sha1 is original commit.

**Comment by openpaas-james** [comment]
Build started sha1 is original commit.

**Comment by chibenwa** [comment]
We have no migration plan.

Honnestly I will not work on this.

**Pull request closed without merging**